mkdocs.yml

```diff
@@ -29,6 +29,7 @@ nav:
       - Connect to Wi-Fi: setup/provision.md
       - Multi-Robot Setup: setup/multi-robot.md
       - ROS 2 on a Pi 4: setup/pi4.md
+      - ROS 2 on a NVIDIA Jetson: setup/jetson.md
     - Create® 3 APIs:
```

Comment:
We may have a merge conflict soon if iRobot's branding PR gets merged before we resolve all the issues in this review. Just a heads up.